### PR TITLE
docs: adding changeloguru projects to next and v1.0.0 content

### PR DESCRIPTION
**Pull request by haunt98**
I am author of [Changeloguru](https://github.com/haunt98/changeloguru). It is a tool to auto generate changelog (`CHANGELOG.md`) from conventional commits.

Example [CHANGELOG](https://github.com/haunt98/changeloguru/blob/main/CHANGELOG.md)

### CODE REVIEW

File: content/next/index.md [hunk 1]
```diff
@@ -242,6 +242,7 @@ Configurable and usable for PHP projects as a composer dependency or usable glob
 * [Stats Builder](https://github.com/MarkFChavez/blox_piece_stats_builder): A stats builder for [Blox Fruits](https://www.roblox.com/games/2753915549/UPDATE-11-Blox-Fruits)
 * [Jenkins X](https://jenkins-x.io/): Jenkins X provides pipeline automation, built-in GitOps, and preview environments to help teams collaborate and accelerate their software delivery at any scale.
 * [Cocogitto](https://github.com/oknozor/cocogitto): Cocogitto is a set of cli tools for the conventional commits and semver specifications.
+* [Changeloguru](https://github.com/haunt98/changeloguru): Auto generate changelog from convetional commits, written in Go.
```
Comment (damianopetrungaro) [inline content/next/index.md:245]:
```suggestion
* [Changeloguru](https://github.com/haunt98/changeloguru): Auto-generate changelog from conventional commits, written in Go.
```

File: content/v1.0.0/index.md [hunk 2]
```diff
@@ -251,6 +251,7 @@ Configurable and usable for PHP projects as a composer dependency or usable glob
 * [Belajarpython](https://github.com/belajarpythoncom/belajarpython.com) Open source Indonesian python programming tutorial site.
 * [Uno Platform](https://platform.uno): Build Mobile, Desktop and WebAssembly apps with C# and XAML. Today. Open source and professionally supported.
 * [Jenkins X](https://jenkins-x.io/): Jenkins X provides pipeline automation, built-in GitOps, and preview environments to help teams collaborate and accelerate their software delivery at any scale.
+* [Changeloguru](https://github.com/haunt98/changeloguru): Auto generate changelog from convetional commits, written in Go.
```
Comment (damianopetrungaro) [inline content/v1.0.0/index.md:254]:
```suggestion
* [Changeloguru](https://github.com/haunt98/changeloguru): Auto-generate changelog from conventional commits, written in Go.
```

**Review by damianopetrungaro — CHANGES_REQUESTED** [review]
Quick one :)

**Comment by haunt98** [comment]
Hi @damianopetrungaro, I have update PR follow your changes, can you please take a look ?

**Pull request merged**